packages/lesswrong/components/users/AccountsResetPassword.jsx

```diff
@@ -19,8 +19,7 @@ AccountsResetPassword.contextTypes = {
   intl: intlShape
 }
 
-AccountsResetPassword.propsTypes = {
-  currentUser: PropTypes.object,
```

Comment:
I think this change will lead to a console warning (on the reset-password page) because `currentUser` is in fact provided (via the `withCurrentUser` HoC), it's just unused.